packages/repository/package.json

```diff
@@ -33,6 +33,7 @@
     "debug": "^4.1.1",
     "lodash": "^4.17.15",
     "loopback-datasource-juggler": "^4.18.1",
+    "openapi3-ts": "^1.3.0",
```

Comment:
As explained in the other comment, we should use `json-schema` types, not `openapi3-ts`.